public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -150,7 +125,10 @@ export class LokiDatasource extends DataSourceApi<LokiQuery, LokiOptions> {
       filter((response: any) => (response.cancelled ? false : true)),
       map((response: { data: LokiResponse }) => {
         if (response.data.data.resultType === LokiResultType.Stream) {
-          throw new Error('Metrics mode does not support logs. Use an aggregation or switch to Logs mode.');
+          return {
+            data: [],
+            key: `${target.refId}_instant`,
```

Comment:
Unclear what's happening here. Also, the `_instant` suffix suggests this has something to do with instant queries.